### PR TITLE
Create docs pages

### PR DESCRIPTION
**Pull request by cpacker**
Viewable at https://memgpt.readthedocs.io

A few minor TODO pages (eg web UI runpod) but otherwise ready to merge.

### CODE REVIEW

**Comment by vivi** [comment]
Can you add docs/README.md that details how to build the docs?

File: docs/adding_wrappers.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Adding Wrappers
```
Comment (vivi) [inline docs/adding_wrappers.md:1]: TODO

File: README.md [hunk 2]
```diff
@@ -300,9 +300,9 @@ While using MemGPT via the CLI (not Discord!) you can run various commands:
 /retry
   pops the last answer and tries to get another one
 /rethink <text>
-  will replace the inner dialog of the last assistant message with the <text> to help shaping the conversation 
+  will replace the inner dialog of the last assistant message with the <text> to help shaping the conversation
 /rewrite
-  will replace the last assistant answer with the given text to correct or force the answer 
+  will replace the last assistant answer with the given text to correct or force the answer
```
Comment (vivi) [inline README.md:305]: These are missing in the docs

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    - 'Supporting new LLM backends': adding_wrappers.md
+    - 'Improving local LLM performance': adding_wrappers.md
+    - 'Contributing to the codebase': contributing.md
+theme: readthedocs
+  # highlightjs: true
+  # hljs_languages:
+  #   - yaml
+  #   - json
+  #   - python
+# include_homepage_in_sidebar: false
```
Comment (vivi) [inline mkdocs.yml:36]: comments?

**Review by vivi — CHANGES_REQUESTED** [review]
remind me to add the [autogen](https://github.com/cpacker/MemGPT/tree/main/memgpt/autogen) docs or (@cpacker if you have time)

File: README.md (hunk 2 above)
Comment (cpacker) [reply README.md:305]: Add on PR

**Comment by cpacker** [comment]
> remind me to add the [autogen](https://github.com/cpacker/MemGPT/tree/main/memgpt/autogen) docs or (@cpacker if you have time)

add on separate PR

**Review by cpacker — COMMENTED** [review]
merging

**Pull request merged**